scratchml/models/svr.py

```diff
@@ -0,0 +1,200 @@
+from abc import ABC
+import numpy as np
+from ..utils import convert_array_numpy
+from ..metrics import (
+    mean_squared_error,
+    root_mean_squared_error,
+    r_squared,
+    mean_absolute_error,
+    median_absolute_error,
+    mean_absolute_percentage_error,
+    mean_squared_logarithmic_error,
+    max_error,
+)
+from ..kernels import linear_kernel, polynomial_kernel, rbf_kernel
+
+
+class BaseSVR(ABC):
```

Comment:
Please, create a `_validate_parameters` function to validate the value of all parameters passed during the model initialization (such as C, kernel, epsilon, degree, and so on).